dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/NodeStats.java

```diff
@@ -0,0 +1,96 @@
+package com.dotcms.content.elasticsearch.business;
+
+import org.elasticsearch.common.unit.ByteSizeValue;
+
+public class NodeStats {
+    private final String name;
+    private final String host;
+    private int size;
+    private String prettySize;
+    private String transportAddress;
+    private boolean master;
+    private int docCount;
+
+    private NodeStats(final Builder builder) {
+        this.name = builder.name;
+        this.host = builder.host;
+        this.size = builder.size;
+        this.prettySize = new ByteSizeValue(size).toString();
+        this.transportAddress = builder.transportAddress;
+        this.master = builder.master;
+        this.docCount = builder.docCount;
+    }
+
+    public String getHost() {
+        return host;
+    }
+
+    public int getSizeRaw() {
+        return size;
+    }
+
+    public String getSize() {
+        return prettySize;
+    }
+
+    public String getTransportAddress() {
+        return transportAddress;
+    }
+
+    public boolean isMaster() {
+        return master;
+    }
+
+    public int getDocCount() {
+        return docCount;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public static class Builder {
+        private String name;
+        private String host;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field host has the same name as a method](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=3712629)